lib/bolt/config.rb

```diff
@@ -31,6 +31,7 @@ def initialize(transport, uri = nil)
     :log,
     :modulepath,
     :puppetdb,
+    :hiera_config,
```

Comment:
We lean towards hyphens rather than underscore for config options. This would be `:'hiera_config'`.

Reply:
I believe the ticket specification was wrong on how to format the option.